### PR TITLE
[16.0][IMP] sale_exception: Improved wizard view and don't ignore exceptions with "is blocking" checked

### PR DESCRIPTION
**Pull request by aliciagaarzo**
After testing the migrated module, I’ve found some improvements:

- Description field is noy properly displayed; user needs to scroll to the right to see it.

- Allow to ignore exceptions even if the “is blocking” option is checked

### CODE REVIEW

**Review by amarcosg — APPROVED** [review]

File: sale_exception/wizard/sale_exception_confirm.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class SaleExceptionConfirm(models.TransientModel):
 
     def action_confirm(self):
         self.ensure_one()
-        if self.ignore:
+        exceptions_blocking = self.exception_ids.filtered(lambda s: s.is_blocking)
```
Comment (rousseldenis) [inline sale_exception/wizard/sale_exception_confirm.py]:
```suggestion
        exceptions_blocking = self.exception_ids.filtered("is_blocking")
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Minor change. Could you add a test for this ?

**Comment by aliciagaarzo** [comment]
Good afternoon @rousseldenis , can you check if the changes are correct?

**Comment by aliciagaarzo** [comment]
@rousseldenis Have you been able to review the changes?

**Review by hugo-cordoba — APPROVED** [review]
LGTM

**Comment by hugo-cordoba** [comment]
@rousseldenis

**Comment by rousseldenis** [comment]
@aliciagaarzo Could you rebase to allow running runboat and test functionally ?

**Comment by aliciagaarzo** [comment]
> @aliciagaarzo Could you rebase to allow running runboat and test functionally ?

Done, thanks for the review!

**Comment by aliciagaarzo** [comment]
@rousseldenis Can you approve the changes after the rebase?

**Comment by glitchov** [comment]
LGTM code & functional

**Comment by aliciagaarzo** [comment]
@rousseldenis

**Pull request merged**